fish_command_timer.fish

```diff
@@ -89,9 +89,9 @@ else if type gdate > /dev/null ^ /dev/null; and gdate +'%N' | grep -qv 'N'
   function fish_command_timer_print_time
     gdate --date="@$argv[1]" +"$fish_command_timer_time_format"
   end
-else if type perl > /dev/null ^ /dev/null; then
+else if type perl > /dev/null ^ /dev/null; and
```

Comment:
Oops, typo from bash :)

The correct syntax would actually be
```
else if type perl > /dev/null ^ /dev/null
```
Could you drop the ";and" ?